Source/stores.cpp

```diff
@@ -1182,7 +1182,7 @@ void S_StartBarMaid()
 {
 	stextsize = false;
 	stextscrl = false;
-	AddSText(0, 2, true, "Gillian", COL_GOLD, false);
+	AddSText(0, 2, true, _("Gillian"), COL_GOLD, false);

```

Comment:
Gillian appears without title, so nothing to translate

```suggestion
	AddSText(0, 2, true, "Gillian", COL_GOLD, false);
```